cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -427,12 +425,16 @@
               Verify that the CWT is a valid CBOR object.
             </t>
             <t>
-              If the object begins with the CWT CBOR tag, remove it
-	      and verify that one of the COSE CBOR tags follows it.
-	    </t>
-	    <t>
-	      If the object is tagged with one of the COSE CBOR tags, remove it
-	      and verify that it corresponds to the structure of the following COSE object.
+              If the object begins with the CWT CBOR tag, remove it and
+              continue verification.
```

Comment:
I believe that the current draft always requires a COSE tag to follow a CWT tag.  The logic was that if you need the CWT tag to know what you're dealing with, you probably also don't have the application context to know what kind of COSE object you're dealing with.  If you disagree with this logic, let me know.

I disagree that a CWT will always be present on the outer-most object.  It's only necessary if the application doesn't already know that the data structure is a CWT.  Normally it will not be needed and we shouldn't require it to be present, as doing so just takes up space and adds parsing.  Maybe I'm misunderstanding what you meant by your last sentence?